runtime/runtime/src/lib.rs

```diff
@@ -1776,7 +1777,7 @@ impl Runtime {
             {
                 processing_state.delayed_receipts.push(
                     &mut processing_state.state_update,
-                    receipt,
+                    receipt.clone(), // TODO get rid of clone
```

Comment:
Leftover TODO

Maybe `StateStoredReceiptV0` could keep a `Cow<Receipt>` to deal with borrowing?